packages/website/default-pages/index.tsx

```diff
@@ -11,16 +11,14 @@ import WidthDetector from '@atlaskit/width-detector';
 
 import { Section } from '../src/components/page';
 import Panel, { PanelGrid } from '../src/components/panel';
-import Meta, { LinkObject } from '../src/components/meta-context';
-import pageInfo from '../src/pages-list';
```

Comment:
I think it makes sense to remove the dependency on pages-list here, and put the required things into meta-context.